static/index.ejs

```diff
@@ -12,9 +12,22 @@
       }
       __sentryMark('head-start');
     </script>
+    <link rel="preload" href="/api/client-config/" as="fetch" crossorigin />
+    <script>
+      // Django inlines a class for the user theme, but SPA mode requires manual init.
+      // Default to `.theme-system` and replace with user theme as soon as possible.
+      const BOOTSTRAP_URL = '/api/client-config/';
+      async function applySpaTheme() {
+        const data = await fetch(BOOTSTRAP_URL).then(res => res.json());
```

Comment:
We need to make sure that we check for res.ok, else json parse will throw an error